packages/mui-varnish/package.json

```
@@ -0,0 +1,68 @@
+{
+  "name": "@allenai/mui-varnish",
+  "version": "5.11.0",
+  "private": false,
+  "author": "Reviz",
+  "description": "Varnish theme and components that implement Google's Material Design.",
+  "main": "./src/index.ts",
+  "keywords": [
+    "react",
+    "react-component",
+    "mui",
+    "material-ui",
+    "material design",
+    "varnish"
+  ],
```

```suggestion
  "keywords": [
    "allenai",
    "varnish"
  ],
```

I don't think we want random folks looking for MUI or react things to use this, but could be wrong